src/components/ParserComponent.tsx

```diff
@@ -132,6 +132,7 @@ export default class ParserComponent extends React.Component<ParserInterface,
                 data,
               };
             });
+            // console.log(this.state.data);
```

Comment:
get rid of these if you're not using them anymore

Reply:
all lines that are code behind comments now deleted